hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieCommitMetadata.java

```diff
@@ -208,6 +208,92 @@ public long fetchTotalWriteErrors() {
     return totalWriteErrors;
   }
 
+  public long fetchTotalRecordsDeleted() {
```

Comment:
rename: getTotalRecordsDeleted() to stay consistent

Reply:
Will do